scripts/actions/__tests__/kitchen-sink.mdx

```diff
@@ -173,5 +180,6 @@ def handler(event, context):
         </tr>
       </tbody>
     </Table>
+
```

Comment:
I think the tests are failing due to the changes in this file. We should revert them.